stytch/b2b/api/organizations_members.py

```diff
@@ -173,12 +173,12 @@ async def delete_async(
         res = await self.async_client.delete(url)
         return DeleteResponse.from_json(res.response.status, res.json)
 
-    def delete_phone_number(
+    def delete_mfa_phone_number(
```

Comment:
Were the old MFA methods published already? Is this a breaking change? (Probably a question for the MFA team)

Reply:
We made these changes last week (pre-launch) but they were only propagated to the Node repo, so I would say this isn't a breaking change